gumstix_raspberry_pi_zero_battery_board_demo/README.md

```diff
@@ -0,0 +1,115 @@
+# Gumstix Raspberry Pi Zero Battery Board Demo
+
+## Download and flash Gumstix’s Raspbian Lite image 
+Download the Gumstix Raspbian Lite image from:
+https://gumstix-raspbian.s3.amazonaws.com/2020-01-12/raspberrypi-cm3/rpi-4.19.y/2019-09-26-raspbian-buster-lite.img.xz
+
+Flash your SD card with the extracted image above. To do this, open a terminal on your host computer. 
+```
+$ sudo dd if=2019-09-26-raspbian-buster-lite.img of=/dev/sdX bs=4MiB
+```
+Where sdX is the location of your SD card (to find the location of the SD card, you can use the command “lsblk” to match the memory device with the SD card). 
+
+## Setting up SSH:
```

Comment:
I take it this is a special SSH/WiFi configuration specific to the app.  Normally there is a wpa_supplicant.conf file on rootfs at /etc for auto-connecting to the network.  It would provide clarity to add a few sentences describing why this data is in /boot, since many users will be semi-experienced linux/rpi devs.

Reply:
Actually, they could just edit the wpa_supplicant.conf file in the /etc/wpa_supplicant/ directory - I will adjust the instructions to simplify this.